<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleContourWidget</class>
 <widget class="QWidget" name="ModuleContourWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSelectColor">
       <property name="text">
        <string>Select Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="pqColorChooserButton" name="colorChooser" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Color By</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbColorBy"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Color Array Minimum</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="tomviz::DoubleSliderWidget" name="sliColorArrayMin" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Color Array Maximum</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Color Map Data</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="cbColorMapData">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="tomviz::DoubleSliderWidget" name="sliValue" native="true"/>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cbRepresentation"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Opacity</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="tomviz::DoubleSliderWidget" name="sliOpacity" native="true"/>
     </item>
     <item row="2" column="1">
      <widget class="tomviz::DoubleSliderWidget" name="sliColorArrayMax" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tomviz::DoubleSliderWidget</class>
   <extends>QWidget</extends>
   <header>DoubleSliderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqColorChooserButton</class>
   <extends>QWidget</extends>
   <header>pqColorChooserButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
